### DISCUSSION TITLE
TTS training using spk avg xvector instead of individual xvector

### DISCUSSION THREAD

**Discussion by pratikkumar018** [espnet/espnet#5008] [Q&A] [answered]
Hi Can you please help how to configure using spk_xvector.scp in training instead of xvector.scp in TTS training.

**Comment by kan-bayashi** [comment, marked as answer]
The most easiest solution is to replace each path of xvector in xvector.scp with one of scp_xvector.scp.

**Reply by pratikkumar018** [reply]
I was also thinking the same. Thanks a lot
